docs/configuration.md

```diff
@@ -1023,6 +1023,24 @@ Apart from these, the following properties are also available, and may be useful
     The max size of event log file before it's rolled over.
   </td>
 </tr>
+<tr>
+  <td><code>spark.eventLog.rolling.maxFilesToRetain</code></td>
+  <td>Int.MaxValue</td>
+  <td>
+    The maximum number of event log files which will be retained as non-compacted.
+    By default, all event log files will be retained. Please set the configuration and
```

Comment:
Might be good to clarify where to configure things, since this config is for the history server, but `spark.eventLog.rolling.maxFileSize` needs to be set in the applications.